messages/compile.json

```diff
@@ -3,5 +3,6 @@
     "propertyFileFlagDescription": "Specify custom property file.",
     "provarHomeFlagDescription": "Specify provar home override",
     "projectPathFlagDescription": "Specify project path override",
-    "loglevelFlagDescription": "Specify log level for command output"
+    "loglevelFlagDescription": "Specify log level for command output",
```

Comment:
I don't want to be a pain but is it possible to sort the JSON strings for messages in alphabetical order so it will be easier to find them in the future?

Reply:
Done